src/core/Modal/Modal/Modal.tsx

```diff
@@ -255,6 +256,7 @@ class BaseModal extends Component<ModalProps> {
     const {
       visible,
       id,
+      propClassName,
```

Comment:
Could we think some more describing name for this `propClassName` to explain it's purpose?
As it does not have JSDoc where to check what it does.

Reply:
What would be more descriptive name? This the className prop provided by the user, but as it is not passed onto the outermost element in the Base component, it needs to be renamed. The Modal component cannot pass className prop to StyledModal, because it would result styled components to add it's own 'hidden' classNames to it. Then the 'hidden' classNames would not end up to the outermost div and styles would break.

The other way would be to use InternalModalProps and have the prop there, but in my opinion it was less elegant solution - this is debatable of course.

Reply:
Based on that explanation, it might hard to give it a describing name. Then the less elegant internal props idea might be good; to have that long explanation for it.
The `Internal*Props` seems to be used in quite many other components so it might be a viable solution.

Reply:
Sure, I can add internal props. Just seemed too much additional boilerplate in my opinion, but if the purpose of the prop is not clear from the naming, it's probably needed.

Reply:
Added internal props with explanation.